### PR TITLE
[WIP] Added 3 columns in Social media section in 1 row.

### PR DESCRIPTION
**Pull request by harshit98**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request
I have added 3 columns in social media section in 1 row. Used a different color for each section. Increased the size of areas to fit well into the design.
Fixes issue no. #924 and #961 
selection_007
selection_009

### CODE REVIEW

**Comment by abishekvashok** [comment]
I would resolve the conflicts for you.... Adding WIP since it's a work in progress.

**Comment by harshit98** [comment]
Screenshot of the following changes I have made in Social Media sections -
screenshot from 2017-01-31 02-14-17

**Comment by abishekvashok** [comment]
@harshit98 i think you got in wrong, just make the sections which were already there smaller don't put it like this.

**Comment by m1guelpf** [comment]
@harshit98 @Abhi2424shek Adding discussion.
Also requesting @robbyoconnor opinion

**Comment by robbyoconnor** [comment]
Please link me to your live site please? You didn't read the text beside those checkboxes...

File: index.html
Comment (robbyoconnor) [inline index.html]: Use a glyphicon/font-awesome icon.

File: index.html
Comment (robbyoconnor) [inline index.html]: Use a glyphicon/font-awesome icon.

File: index.html
Comment (robbyoconnor) [inline index.html]: The file name is absurd. Use a glyphicon/font-awesome icon instead. We load too many images as-is.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
- Use icons and not images 
- if you insist on using the images...Actually be mindful of their names! 

Your file names are absurd. Rename them, or better yet don't even use them at all! Use either glyphicon or font-awesome.

**Comment by robbyoconnor** [comment]
@fossasia/gci16-maintainers -- do not merge this. The file names for images chosen by @harshit98 are bad, really bad.

**Comment by harshit98** [comment]
@robbyoconnor Okay, I'll try to fix it as per your suggestions.

**Comment by harshit98** [comment]
Live Link Preview - https://harshit98.github.io/gci16.fossasia.org/

**Comment by robbyoconnor** [comment]
@harshit98 Here's what you need to do: 

- Clone your fork
- Work locally
- **_DELETE_** The images entirely.

**Comment by robbyoconnor** [comment]
@harshit98 Here's what you need to do: 

- Clone your fork
- Work locally
- **_DELETE_** The images entirely. 
- Commit and push! 
- ???
- PROFIT1111!111

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
Looks Better...Now squash.

**Comment by harshit98** [comment]
@robbyoconnor okay.

File: index.html
Comment (robbyoconnor) [inline index.html]: Move comment to next line

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
When coding be consistent with style.  See code comments

**Comment by robbyoconnor** [comment]
Fix linting issues found by Coala please: https://travis-ci.org/fossasia/gci16.fossasia.org/builds/197093160#L450

**Comment by robbyoconnor** [comment]
You have got to be kidding me. **_WORK LOCALLY ON YOUR COMPUTER!_**

**Comment by robbyoconnor** [comment]
Run coala locally: Instructions are here: http://coala.io/#!/home

**Comment by robbyoconnor** [comment]
The the style violations you are seeing -- look at the surrounding code and how things are done.

**Comment by robbyoconnor** [comment]
**EDIT THE FILES ON YOUR COMPUTER AND USE MEANINFUL COMMIT MESSAGES!**

**Comment by robbyoconnor** [comment]
@harshit98  -- **EDIT THE FILES ON YOUR COMPUTER AND USE MEANINFUL COMMIT MESSAGES!**

**Comment by harshit98** [comment]
I have edited the file on my computer only. Also, runned the coala locally - the command you gave it worked. And fixed the linting errors.

**Comment by robbyoconnor** [comment]
Okay then fix your commit messages. They're horrible.

**Comment by robbyoconnor** [comment]
@fossasia/gci16-maintainers: what's your opinion?

**Comment by abishekvashok** [comment]
Well more work needed:
screenshot_2017-02-03-09-33-33-226_com android chrome


Not responsive design

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
- [ ] You
 still have responsiveness problems. 

Here's what you need to do to test: narrow your browser window...things should reflow naturally... This doesn't happen here. Narrowing your browser window will simulate what happens on a cell phone.

**Comment by harshit98** [comment]
@robbyoconnor I did what you said. I didn't saw any overlapping of sections as per screenshot by @Abhi2424shek

**Comment by robbyoconnor** [comment]
@harshit98 -- It just plain doesn't look good.

**Comment by robbyoconnor** [comment]
The +

**Comment by robbyoconnor** [comment]
here: 

image

image

**Comment by robbyoconnor** [comment]
No overlapping?

**Comment by harshit98** [comment]
@robbyoconnor ok. But now, I'm not doing this. Please close this PR.

**Comment by robbyoconnor** [comment]
Why are you giving up? Because it's not easy? Consider a better design that works in a way that's responsive for ANY screen size.

**Comment by abishekvashok** [comment]
Dont give up

**Comment by harshit98** [comment]
@robbyoconnor any suggestions on responsive design?

**Comment by robbyoconnor** [comment]
I'm not the best person.. but what you're doing isn't the correct way. Read up on responsive design.

**Comment by harshit98** [comment]
So should I keep the design same and just make some tweaks so that the section is responsive for every screen size?

**Comment by harshit98** [comment]
Okay, I'll implement a new design which will be responsive. Should I remove the flicker slider? It is creating a lot of trouble! and slow time while loading the social-media section. Instead of using the flicker-slider, I'll add the section of flicker with an icon and link, which will be a nice way and easier to load the social-section. And that too, easier to see a clean design on any phone screen size!

**Comment by robbyoconnor** [comment]
No. Everything should stay.

**Comment by harshit98** [comment]
Live Link Preview - https://harshit98.github.io/gci16.fossasia.org/

**Comment by m1guelpf** [comment]
screenshot_2017-02-05-16-43-54

**Comment by ankitrgadiya** [comment]
No offence but TBH this doesn't make sense to me. We're already having fa icons in the footer. This section was supposed to show iframe or something like that using social media api.

**Comment by robbyoconnor** [comment]
Yeah...I'm with @ankitrgadiya on this..

**Comment by robbyoconnor** [comment]
@harshit98 -- look at the current implementation and just change the layout using Bootstrap.

**Comment by harshit98** [comment]
Sorry did the wrong commits. Close this PR, I'll make a new one.

**Pull request closed without merging**
